docker-compose.yml

```diff
@@ -2,26 +2,26 @@ version: "3.9"
 services:
   mysql:
     env_file: [.env_file]
-    image: mysql:latest
+    image: mysql:8.0.33
```

Comment:
If it works now it doesn't mean it will work tomorrow.
I think there are several different approaches towards docker container versioning, but here's the one definitely used at least for some DB images:
Version named `latest` **does not** correspond to the latest version of DB. It only means that this version was most recently published to Docker repo. It can be latest or any previous supported and updated version.
So here (as far as I understood) we can't really rely on "latest".